errors/messages.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+const (
+	ErrMsgDomain = "dapr.io"
+
+	errStringFormat = "api error: code = %s desc = %s"
+
+	typeGoogleAPI = "type.googleapis.com/"
+
+	// Messages
+	MsgStateGet        = "failed to get %s from state store %s: %s"
```

Comment:
Do these constants need to be exported? Since they are formatting consts, should they be private?

Reply:
we are using these in dapr

Reply:
Ok please add a comment that these are used in Dapr.

However consider whether keeping them in dapr/kit is the right place. If you need to add or update constants, having them in a separate repo will make your life a lot harder

Reply:
Initially we thought we might use these in both dapr and components-contrib, but for now I'll rm them and add them back if we find the need to use the same messages in both dapr and components-contrib

Reply:
rm-ing them